tests/featureWriters/kernFeatureWriter_test.py

```diff
@@ -720,30 +753,40 @@ def test_kern_LTR_and_RTL(self, FontClass):
             @kern1.reh = [reh-ar zain-ar reh-ar.fina];
             @kern2.alef = [alef-ar alef-ar.isol];
 
-            lookup kern_dflt {
-                pos seven four -25;
-            } kern_dflt;
-
-            lookup kern_ltr {
-                enum pos @kern1.A V -40;
-            } kern_ltr;
-
-            lookup kern_rtl {
+            lookup kern_Arab {
                 pos four-ar seven-ar -30;
                 pos reh-ar.fina lam-ar.init <-80 0 -80 0>;
                 pos @kern1.reh @kern2.alef <-100 0 -100 0>;
-            } kern_rtl;
+            } kern_Arab;
+
+            lookup kern_Latn {
+                enum pos @kern1.A V -40;
+            } kern_Latn;
+
+            lookup kern_Common {
+                pos seven four -25;
+            } kern_Common;
 
             feature kern {
-                lookup kern_dflt;
+                script DFLT;
+                language dflt;
+                lookup kern_Common;
+                script arab;
+                language dflt;
+                lookup kern_Arab;
+                language URD;
                 script latn;
                 language dflt;
-                lookup kern_ltr;
+                lookup kern_Common;
                 language TRK;
                 script arab;
                 language dflt;
-                lookup kern_rtl;
+                lookup kern_Common;
                 language URD;
+                script latn;
+                language dflt;
+                lookup kern_Latn;
```

Comment:
it's not clear in what order the various lookups are registered under each feature's scripts and languages, I would have expected that, say, all the lookups for the combo latn/dflt would be grouped together instead of having multiple latn/dflt declarations, one with `kern_Common` a few lines above, then this one for `kern_Latn`, interspersed with other different script/langs declarations.
probably doesn't matter in the end, but would aid readability/debugging if the all the lookup references for a given script/langs combo would be consecutive

Reply:
@simoncozens it's much nicer to read now, thanks!